.storybook/typings.d.ts

```diff
@@ -1,6 +1,13 @@
 /* eslint-disable */
 // @ts-nocheck
 
+declare const global: {
```

Comment:
if you rebase this branch with the feature branch, these changes should be applied and should not appear in the PR anymore

Reply:
rebased